SimpleTotp.Tests/Base32ConvertTests.cs

```diff
@@ -0,0 +1,174 @@
+﻿// ReSharper disable ExpressionIsAlwaysNull
+// ReSharper disable ConditionIsAlwaysTrueOrFalse
+// ReSharper disable ConvertToConstant.Local
+// ReSharper disable StringLiteralTypo
+
+using System;
+using System.Diagnostics.CodeAnalysis;
+using System.Text;
+using Xunit;
+
+namespace SimpleTotp.Tests
+{
+    [ExcludeFromCodeCoverage]
```

Comment:
Why?

Reply:
Why would you want to measure code coverage of the test class itself?